packages/sources/bitgo/src/config/index.ts

```diff
@@ -0,0 +1,15 @@
+import { AdapterConfig } from '@chainlink/external-adapter-framework/config'
+
+export const config = new AdapterConfig({
+  API_LIMIT: {
+    description: 'The default maximum number of results to request from the API',
+    type: 'number',
+    default: 100,
+  },
+  BACKGROUND_EXECUTE_MS: {
```

Comment:
Sucks that this doesn't translate as well at the [framework level]( https://github.com/smartcontractkit/ea-framework-js/blob/main/src/config/index.ts#L327-L345), but this seems fine.